package.json

```diff
@@ -24,9 +24,7 @@
             {
                 "id": "vividus-story",
                 "aliases": [
-                    "bdd",
-                    "jbehave",
-                    "dsl"
```

Comment:
why not to keep old aliases?

Reply:
not sure if we really need them, we aren't bdd, we do not support jbehave, dsl also looks redundant

Reply:
Yet you're using story :) What refers to BDD :)